src/style.css

```
@@ -1,3 +1,67 @@
+:root {
+  --accent-color: #00acdc;
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: black;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 23px 0;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+}
```

just add list-style: none to `.nav__list`, and remove for nav__item
```suggestion
.nav__list {
  display: flex;
  padding: 0;
  margin: 0;
}